api/schemas/serializers.py

```diff
@@ -57,11 +52,13 @@ def get_absolute_url(self, obj):
     class Meta:
         type_ = 'form_blocks'
 
+
 class RegistrationSchemaSerializer(SchemaSerializer):
+    description = ser.CharField(read_only=True, allow_blank=True)
+
     form_blocks = RelationshipField(
         related_view='schemas:registration-schema-form-blocks',
         related_view_kwargs={'schema_id': '<_id>'},
-        always_embed=True,
```

Comment:
if `always_embed` is True, then front end needs to be able to handle form_blocks - we can restore this later when front end is ready.